ecp5/arch.h

```diff
@@ -614,21 +624,46 @@ struct Arch : BaseArch<ArchRanges>
 
     uint32_t getWireChecksum(WireId wire) const override { return wire.index; }
 
+    uint32_t getWireVecIdx(const WireId & e) const {
```

Comment:
by convention, functions not part of the standard arch API should have a `snake_case` name like `get_wire_vec_idx`

Reply:
OK yep, fix pushed. Better name suggestions than `vecidx` are welcome :)